### ISSUE TITLE
Action "Turn monitor on" shifts cursor

### ISSUE THREAD

**Issue by maffe** [EventGhost/EventGhost#410]
https://github.com/EventGhost/EventGhost/blob/177be516849e74970d2e13cda82244be09f277ce/plugins/System/__init__.py#L122
I see [why](https://stackoverflow.com/questions/12572441/sendmessage-sc-monitorpower-wont-turn-monitor-on-when-running-windows-8) this is done, but please move my cursor back :-) I had this action triggered multiple times and it moved the cursor away from the position I remembered, the following blind click did something unintended :(

**Comment by kdschlosser** [comment]
fair enough. Never thought that the mouse moving a single pixel would cause a problem. But here it is.. I am sorry for any issues you had because of this. You are more then welcome to make the changes needed and submit a pull request for the changes made. I would appreciate you doing that. I like to keep a record of the prople who have spent the time to locate a big and find a solution.
